lib/ReactViews/Map/Navigation/AugmentedVirtualityTool.jsx

```diff
@@ -36,6 +36,33 @@ class AugmentedVirtualityTool extends React.Component {
   handleClickAVTool() {
     // Make the AugmentedVirtuality module avaliable elsewhere.
     this.props.terria.augmentedVirtuality = this.augmentedVirtuality;
+    // feature detect for new ios 13
+    // it seems you don't need to ask for both, but who knows, ios 14 / something
+    // could change again
+    if (
+      DeviceMotionEvent &&
```

Comment:
This should probably be `window.DeviceMotionEvent`  My chrome version doesn't have DeviceMotionEvent and it errors out saying DeviceMotionEvent is undefined.

Same with `DeviceOrientationEvent` below.

Reply:
Thanks Nanda this is a good catch, I was too focused on whether `requestPermission` existed!